http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/PersonResource.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.smallrye.mutiny.Uni;
+
+@Path("/persons")
+public class PersonResource {
+
+    private final Set<Person> personSet = Collections.synchronizedSet(new LinkedHashSet<>());
+
+    @Path("/process-json")
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Uni<Response> processJson(String jsonData) {
```

Comment:
@jcarranzan @jedla97 TP https://github.com/quarkus-qe/quarkus-test-plans/blob/main/QQE-379.md?plain=1#L31 says `Deserialization: Ensure accurate conversion of JSON payloads, that it's performed using the configured JSON libraries (quarkus-rest-jackson, quarkus-rest-jsonb).` 

I don't see any deserialization performed by `quarkus-rest-jackson` or `quarkus-rest-jsonb`. AFAICT it is always performed manually. Am I wrong?